.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Checklist:
+ - Write / update unit tests
+ - Write / update integration (envtest) tests
+ - Ensure there is an issue for e2e tests if needed
+ - Make sure `make test` passes
+ - Test code changes manually
+ - Update readme and documentation
+ - Write PR description that highlights overall changes and add usage examples if applicable
+-->
```

Comment:
Add an additional item for test coverage?

Reply:
I think it could be covered by the tests items above

Reply:
+1, test coverage did not decrease